certified-connectors/Advanced Data Operations/README.md

```diff
@@ -0,0 +1,38 @@
+# Advanced Data Operations
+
+A suite of advanced API's designed to assist in the transformation of bulk datasets.
+
```

Comment:
@brad302 please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1).

Reply:
Thanks @v-EgorKozhadei, that's done.  I hope it now confirms.